src/components/presentational/NotesInput/NotesInput.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useRef, useEffect } from 'react';
+import { format } from 'date-fns';
+import "./NotesInput.css"
+import language from '../../../helpers/language';
+
+export interface NotesInputProps {
+	autoTimestamp?: boolean;
+    placeholder?: string;
+	onChange(value: string): void;
+	value: string;
```

Comment:
Do we need to require a value?

Reply:
Yeah this is a controlled component which I think is the pattern normally used for inputs etc...